src/Catty/IO/ProjectManager.swift

```diff
@@ -0,0 +1,109 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+@objc(ProjectManager)
+@objcMembers class ProjectManager: NSObject {
+
+    static func createProject(name: String, projectId: String?) -> Project {
+        ProjectManager.createProject(name: name, projectId: projectId, fileManager: CBFileManager.shared())
+    }
+
+    static func createProject(name: String, projectId: String?, fileManager: CBFileManager) -> Project {
+
+        let project = Project()
+        let projectName = Util.uniqueName(name, existingNames: Project.allProjectNames())
+        project.scene = Scene(name: "Scene 1")
```

Comment:
Please use the localized `defaultSceneNameForSceneNumber` method in `Util` instead of hardcoding the scene name here.